vars/dockerExecute.groovy

```diff
@@ -120,16 +122,18 @@ void call(Map parameters = [:], body) {
             .loadStepDefaults()
             .mixinGeneralConfig(script.commonPipelineEnvironment, GENERAL_CONFIG_KEYS)
             .mixinStepConfig(script.commonPipelineEnvironment, STEP_CONFIG_KEYS)
-            .mixinStageConfig(script.commonPipelineEnvironment, parameters.stageName?:env.STAGE_NAME, STEP_CONFIG_KEYS)
+            .mixinStageConfig(script.commonPipelineEnvironment, parameters.stageName ?: env.STAGE_NAME, STEP_CONFIG_KEYS)
             .mixin(parameters, PARAMETER_KEYS)
             .use()
 
+        SidecarUtils sidecarUtils = new SidecarUtils(script)
+
         new Utils().pushToSWA([
-            step: STEP_NAME,
+            step         : STEP_NAME,
```

Comment:
I think usually we do not add additional spaces to align the values

Reply:
done